docs/mine/lotus/split-markets-miners.md

```diff
@@ -126,3 +126,41 @@ export LOTUS_MINER_PATH=~/markets-repo-location
 ```sh
 LOTUS_MINER_PATH=~/markets-repo-location ./lotus-miner run
 ```
+
+## Rollback from a multi-process architecture back to `lotus-miner` monolith process
+
+In case you want to revert the changes listed above and go back to running `lotus-miner` as a single process, you have to do the following:
+
+1. Make sure that the `mining/sealing/proving` node is publicly exposed, as we will be enabling the markets subsystem on it.
+
+2. In the `mining/sealing/proving` repository, update the `config.toml` and set `EnableMarkets` option to `true`
+
+```
+[Subsystems]
+  EnableMining = true
+  EnableSealing = true
+  EnableSectorStorage = true
+  EnableMarkets = true
+```
+
+
+3. Restart the `mining/sealing/proving` node (with the default LOTUS_MINER_PATH, which should point to your `mining/sealing/proving` node repo). Note that `lotus-miner` interacts with a given repository depending on the `LOTUS_MINER_PATH` environment variable!
+
+```sh
```

Comment:
````suggestion
```shell
````

It's better to use full language names, rather than their aliases. It makes visually scanning the docs easier for writers.